### ISSUE TITLE
[DOCS] "Contributing" page sets unrealistic expectations

### ISSUE THREAD

**Issue by pjcreath** [saltstack/salt#62498]
**Description**
The [CONTRIBUTING page](https://github.com/saltstack/salt/blob/master/CONTRIBUTING.rst#now-what) claims that PRs with test cases will generally receive some kind of response the same business day, and sometimes even been merged that day.

**Suggested Fix**
The documentation should be updated to reflect the kind of response that contributors should expect, rather than presenting such an optimistically ambitious standard.

Also, what is the preferred way of kicking loose an issue that seems to have fallen through the cracks?  How long should one wait before pinging an assigned reviewer if we haven't heard from them?

**Type of documentation**
CONTRIBUTING guide.

**Location or format of documentation**
https://github.com/saltstack/salt/blob/master/CONTRIBUTING.rst#now-what

**Additional context**
n/a

**Comment by barbaricyawps** [comment]
Hi, there. I'm responding to this issue because you added the Documentation label to this issue, which makes it so that I eventually see it when I do docs triage.

First off, I want to acknowledge the frustration you possibly feel about having your PR languish over time. That's a valid source of frustration.

I think probably the core Salt team and community should consider having some discussions about what expectations are appropriate to have and what options community members should have if they want to get their PR some more attention. It's an area where I think we could definitely do better communication-wise and expectation-wise. This could possibly be a good discussion item for Community Open Hour too.

**Comment by pjcreath** [comment]
Thank you for your thoughtful and considerate response!

I've been on the other side of this, so I'm not actually frustrated that the team can't get back to me same-day.  I just wanted the docs to be update to reflect reality.  :)

The Community Open Hour is news to me, so maybe that's something that can be part of a documentation update too...

**Comment by barbaricyawps** [comment]
Agreed! I would also love to see us offer Open Hours at a variety of times that are EMEA or APAC friendly, not just Americas friendly.

Open Hours are usually held every 1st and 3rd Thursday from 10a.m. to 11a.m. Pacific Time. You can get the link from the [Community Calendar](https://outlook.office365.com/calendar/published/284d7da9ea784ea28c793573dc6ae778@vmware.com/f9fac9fdab1c4ce0b72f5149cb5008b115345329240750421486/calendar.html) or from the [Community Slack workspace ](https://saltstackcommunity.herokuapp.com/).

:point_up: Seems like that should also go into the guide.

I'm going to leave the Needs Triage label on this so that it gets picked up in a future Triage meeting with the team. I'm also going to tag @waynew who wrote the most recent iteration of the guide for his awareness.

**Comment by barbaricyawps** [comment]
I wanted to follow up on this issue to let you know that I shared this issue with our Community Manager and he's been having some great conversations with the team about how we can improve our response times, set reasonable SLAs, and then communicate SLA expectations. We're working on it! 😄
